srcpkgs/pam_ssh_agent_auth/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'pam_ssh_agent_auth'
+pkgname=pam_ssh_agent_auth
+version=0.10.3
+revision=1
+archs="i686 x86_64"
```

Comment:
Architectures shouldn't be limited unless there is a good reason i.e. depending on architecture features/assembler.

Reply:
As it failed to cross-compile locally, I limited the archs. I didn't think this was problematic, however, now that you mentionned the custom LD=gcc variable, seems like it was for the best. What should be done here?